### DISCUSSION TITLE
[1주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#11] [General]
### 진행순서
~ 20:15 PR리뷰
~ 20:40 발표
~ 20:50 회고 (좋았던 점, 건의사항, 느낀점 등..)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [Database] RDBMS와 NoSQL #5 
2. 이주형
 * [Spring] 다양한 의존관계 주입 #7
3. 김정윤
 * [OS] 프로그램 vs 프로세스 vs 스레드 #10

**Comment by ChaerinYu** [comment]
### 회고
* 좋았던 점
* 건의사항
* 느낀점

**Reply by ChaerinYu** [reply]
* 느낀 점
첫 스터디인데 모두 준비 잘해주셔서 감사합니다 ㅠㅠ

**Reply by KJY97** [reply]
- 좋았던 점
   CS 한 분야에 대해 정리할 때 많은 시간이나 조사가 필요한데, 같이 내용 공유하고 정리해서 너무 좋았어요.

**Reply by leeejuhyeong** [reply]
- 좋았떤 점
내용 공유라서 가독성을 위해 정리하게되다보니 더 꼼꼼해지는 것같아서 좋아요
- 느낀점
마크다운 오류잡고 발표준비를 더 해오갰습니다
